app/templates/main/checkboxes.html

```diff
@@ -1,18 +1,20 @@
 {% macro checkboxItems(name, items) %}
 <div class="govuk-form-group">
     <fieldset class="govuk-fieldset" aria-describedby="">
-        <div class="govuk-checkboxes" data-module="govuk-checkboxes">
-            {% for item in items %}
-            {% if item %}
-            <div class="govuk-checkboxes__item">
-                <input class="govuk-checkboxes__input" id="{{ name }}-{{ item['id'] }}" name="{{ name }}" type="checkbox"
-                       value="{{ item['id'] }}">
-                <label class="govuk-label govuk-checkboxes__label" for="{{ name }}-{{ item['id'] }}">
-                    {{ item['name'] }}
-                </label>
+        <div class="scrollable-checkboxes">
+            <div class="govuk-checkboxes" data-module="govuk-checkboxes">
```

Comment:
Have we tried adding this class as an additional class on the div below?

Reply:
No, I'll try that, and refactor it if successful.